client/app/actions/nav/nav.jest.spec.js

```diff
@@ -0,0 +1,12 @@
+/* global test */
```

Comment:
Instead of a file name `nav.jest.spec.js` what about sticking with `foo.spec.js`?  Any reason to differentiate jest tests from other tests? If there is what about `foo.test.js` (another well used nomenclature)

Reply:
@AllenBW  If we did what you were suggesting  then the karma test runner would pick the tests up and they would not work being run through karma.  The tests have to be run through Jests test runner.  I know its an annoyance but the easiest way I could come up with for the two test runners to coexist peacefully.

Reply:
Yeah I thought as much so can we go with `.test` or something equally as obvious, but not specific to the tool?

Reply:
lookin' at the `karma.conf` `.test.js` should work as its only set to pickup on `.spec.js`
https://github.com/AllenBW/manageiq-ui-service/blob/e7c13af07adea8b5f217925697714c878bd1b3cf/karma.conf.js#L45